app/helpers/users_helper.rb

```diff
@@ -62,20 +48,87 @@ def redirect_back_or(default)
   def store_location
     session[:forwarding_url] = request.original_url if request.get?
   end
+  def is_admin?
+    if !current_user.nil? &&current_user.admin
+      return true
+    end
+    return false
 
+  end
 
-  # Returns true if the user is logged in, false otherwise.
   def is_access?
-    @review = Review.find(params[:id])
+    @review = Review.find(params[:review_id])
     if(!@review.nil?)
-       @review_user =  User.find_by(id: @review.user_id)
-     end
+      @review_user =  User.find_by(id: @review.user_id)
+    end
     if((!current_user.nil? && (@review.user_id ==current_user.id)||(!@review_user.nil? && @review_user.manager_id==current_user.id)))
       return true
     end
     return false
+  end
+
+  def is_manager_for_this_review?
+    @review = Review.find(params[:review_id])
+    if(!@review.nil?)
+      @review_user =  User.find_by(id: @review.user_id)
+    end
+    if((!current_user.nil? && (!@review_user.nil? && @review_user.manager_id==current_user.id)))
+      return true
+    end
+    return false
+  end
+
+  def is_manager?
+    if (!current_user.nil? && current_user.manager)
+      return true
```

Comment:
@chandu1307  No need of if and else here.
as current_user.manager is boolean you can just write on line for this method as:
!current_user.nil? && current_user.manager?